src/xcode/ENA/ENA/Source/Scenes/TraceLocations/TraceLocationsCoordinator.swift

```diff
@@ -168,54 +168,76 @@ class TraceLocationsCoordinator {
 				isSecondaryButtonHidden: false
 			)
 		)
-
+		
 		let topBottomContainerViewController = TopBottomContainerViewController(
 			topController: traceLocationDetailsViewController,
 			bottomController: footerViewController
 		)
-
+		
 		traceLocationDetailsNavigationController = UINavigationController(rootViewController: topBottomContainerViewController)
 		parentNavigationController?.present(traceLocationDetailsNavigationController, animated: true)
 	}
-
+	
 	private func showPrintVersionScreen(traceLocation: TraceLocation) {
 		let viewController = TraceLocationPrintVersionViewController(
 			viewModel: TraceLocationPrintVersionViewModel(traceLocation: traceLocation)
 		)
-
+		
 		traceLocationDetailsNavigationController.pushViewController(viewController, animated: true)
 	}
-
+	
 	private func showTraceLocationTypeSelectionScreen() {
+		
+		let locations: [TraceLocationType] = [
+			.locationTypePermanentRetail,
+			.locationTypePermanentFoodService,
+			.locationTypePermanentCraft,
+			.locationTypePermanentWorkplace,
+			.locationTypePermanentEducationalInstitution,
+			.locationTypePermanentPublicBuilding,
+			.locationTypePermanentOther
+		]
+		
+		let events: [TraceLocationType] = [
+			.locationTypeTemporaryCulturalEvent,
+			.locationTypeTemporaryClubActivity,
+			.locationTypeTemporaryPrivateEvent,
+			.locationTypeTemporaryWorshipService,
+			.locationTypeTemporaryOther
+		]
```

Comment:
Not sure if this needs to be configurable from the outside, I think I would prefer to have that information inside the view model.

Reply:
it will make things it more flexible - will keep it here.